platformio.ini

```diff
@@ -139,6 +139,27 @@ build_flags = ${env.build_flags}
     -DHOYMILES_PIN_CE=4
     -DHOYMILES_PIN_CS=5
 
+[env:seeed_xiao_esp32c3]
+; https://wiki.seeedstudio.com/XIAO_ESP32C3_Getting_Started
+board = seeed_xiao_esp32c3
+build_flags = ${env.build_flags}
+    -DHOYMILES_PIN_MISO=9
+    -DHOYMILES_PIN_MOSI=10
+    -DHOYMILES_PIN_SCLK=8
+    -DHOYMILES_PIN_IRQ=3
+    -DHOYMILES_PIN_CE=4
+    -DHOYMILES_PIN_CS=5
+
+[env:esp32_c3_devkitm_1]
+platform = espressif32
+board = esp32-c3-devkitm-1
+build_flags = ${env.build_flags}
+    -DHOYMILES_PIN_CS=2
+    -DHOYMILES_PIN_MOSI=6
+    -DHOYMILES_PIN_MISO=5
+    -DHOYMILES_PIN_SCLK=4
+    -DHOYMILES_PIN_IRQ=0

```

Comment:
It is imho very dangerous to set the IRQ to gpio0. If the IRQ triggers for some reasons during the bootup the ESP will enter the flash mode.

Reply:
that is a very good point, although it seems for the C3, GPIO0 is not a strapping pin for that chip according to Espressif docs:
![grafik](https://user-images.githubusercontent.com/25175069/228915938-17748b28-8b91-405d-ba7c-9f7b856e4f3e.png)

https://docs.espressif.com/projects/esp-idf/en/v5.0/esp32c3/hw-reference/esp32c3/user-guide-devkitm-1.html

Reply:
You are right... my fault. Its gpio9 for these devices

Reply:
I also got confused there for a moment...kind of weird Espressif hasn't established a convention for this.